src/links.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import type { Plugin } from 'unified';
+import type { Root } from 'mdast';
+import type { Link } from 'myst-spec';
+import { selectAll } from 'unist-util-select';
+import type { StaticNotebook } from '@jupyterlab/notebook';
+import { URLExt } from '@jupyterlab/coreutils';
+import type { LinkProps } from '@myst-theme/providers';
+import { IRenderMime } from '@jupyterlab/rendermime';
+
+/**
+ * Handle an anchor node.
+ * NOTE: This is copied from @jupyterlab/rendermime renderers.ts
+ * ideally this should be removed and exported from there?
+ */
+function handleAnchor(
+  anchor: HTMLAnchorElement,
+  resolver: IRenderMime.IResolver,
+  linkHandler: IRenderMime.ILinkHandler | null
+): Promise<void> {
```

Comment:
I pulled this from inside of here:
https://github.com/jupyterlab/jupyterlab/blob/2e8677624ed7301d47de703570a14ce3bb8036cf/packages/rendermime/src/renderers.ts#L950

I am not sure the best way to get at it otherwise!?!